nameko/messaging.py

```diff
@@ -78,19 +76,23 @@ class Publisher(DependencyProvider, HeaderEncoder):
     def __init__(self, exchange=None, queue=None, **defaults):
         """ Provides an AMQP message publisher method via dependency injection.
 
-        In AMQP messages are published to *exchanges* and routed to bound
-        *queues*. This dependency accepts either an `exchange` or a bound
-        `queue`, and will ensure both are declared before publishing.
+        In AMQP, messages are published to *exchanges* and routed to bound
+        *queues*. This dependency accepts the `exchange` to publish to and
+        will ensure that it is declared before publishing.
+
+        Optionally, you may use the `declare` keyword argument to pass a list
```

Comment:
Ah yes, this is definitely required. I think it's probably best to mention ``defaults`` in the docstring and reference the kombu publish method. Thanks